__test__/App.spec.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { BrowserRouter, Switch } from 'react-router-dom';
+import thunk from 'redux-thunk';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import LoginPage from '@pages/loginPage';
+import HomePage from '@pages/homePage';
+import NotFoundPage from '@pages/notFoundPage';
+import Menu from '../src/components/userMenu';
+import Footer from '../src/components/Footer/index';
+import Auth from '../src/components/auth';
+
+import App from '../src/routes/AppRouter';
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+const store = mockStore({
+  authReducer: {
+    isAuthenticated: false,
+    user: {},
+    status: 'rest'
+  }
+});
+
+const auth = (state = {}, action) => {
+  switch (action.type) {
+    case 'LOGIN_SUCCESS':
+      return {
+        isLoginPending: false,
+        isAuthenticated: !action.user,
+        user: action.user
+      };
+    default:
+      return state;
+  }
+};
+
+describe('Application test', () => {
+  it('should render index page', () => {
+    const comp = (
+      <Provider store={store}>
+        <BrowserRouter>
+          <Switch>
+            <App dispatch={jest.fn()} />
+          </Switch>
+        </BrowserRouter>
+      </Provider>
+    );
+    const wrapper = mount(comp);
+
+    expect(wrapper.find('Home')).toBeTruthy();
+  });
+  it('should not crash app', () => {
+    const wrapper = mount(
```

Comment:
this can be "should render the page without crashing", the message does not state if the page was rendered or not.